js/src/widgets/layersTab.js

```diff
@@ -62,6 +64,32 @@
       this.localState(localState);
```

Comment:
Also, before merging, the layers tab should not be visible to the user for non-ImageView modules.